src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -176,18 +176,18 @@ def generate_FAD_2_default_config(self, filename):
     }
     # Generate FAD Records for each record type like cbsd,zone and esc_sensor
     cbsd_fad_records = generateCbsdRecords(cbsd_records, grant_record_list)
-    ppa_fad_record = generatePpaRecords(ppa_records, cbsd_reference_ids)
-    esc_fad_record = [esc_sensor]
+    ppa_fad_records = generatePpaRecords(ppa_records, cbsd_reference_ids)
+    esc_fad_records = [esc_sensor]
 
     sas_harness_dump_records = {
         'cbsdRecords': cbsd_fad_records,
-        'ppaRecord': ppa_fad_record,
-        'escSensorRecord': esc_fad_record
+        'ppaRecords': ppa_fad_records,
+        'escSensorRecords': esc_fad_records
     }
     config = {
         'registrationRequestC2': device_c2,
         'registrationRequestC4': device_c4,
-        'conditionals': conditional_parameters,
+        'conditionalRegistrationData': conditionals,
         'grantRequestG2': grant_g2,
         'grantRequestG4': grant_g4,
         'palRecords': pal_records[0],
```

Comment:
Why are you including only the first element? This should be a list.

Reply:
Updated   and entire  pal_record will be passed   to InjectPalDataBaseRecord